src/partial-result-stream.ts

```diff
@@ -50,11 +50,16 @@ interface RequestFunction {
  *     that it is not ready for any more data. Increase this value if you
  *     experience 'Stream is still not ready to receive data' errors as a
  *     result of a slow writer in your receiving stream.
+ * @property {object} [columnInfo] An object map using which we can pass
+ * additional properties for each column type which can help in deserializing
+ * the data coming from backend. (Eg: We need to pass Proto Function and Enum
+ * map to deserialize proto messages and enums respectively)
```

Comment:
Could this be made more concrete? When do I use it? How do I use it?

Reply:
I'm also confused by what this is and what it's meant to look like. An example in the doc string might help.